lighthouse-core/computed/main-thread-tasks.js

```diff
@@ -133,6 +138,12 @@ class MainThreadTasks {
       }
     }
 
+    while (currentTask && !Number.isFinite(currentTask.endTime)) {
+      // The last event didn't finish before tracing stopped, just end it when tracing ended.
+      currentTask.endTime = lastTraceTs;
+      currentTask = currentTask.parent;
+    }
+
```

Comment:
Lets add a comment here with our expectation like

    // At this point we expect all tasks to have a finite startTime and endTime.